src/Generator/Driver.cs

```diff
@@ -29,6 +29,7 @@ public class Driver
         public Driver(DriverOptions options)
         {
             Options = options;
+            Options.GenerateSequentialLayout = true;
```

Comment:
This is completely adequate but it would be simpler and shorter to move it to the definition of the property.